apps/desktop/src/native/core/peaks.cpp

```
@@ -0,0 +1,36 @@
+#include "core/peaks.hpp"
+
+#include <cmath>
+
+namespace shiranami::audio {
+
+void reducePeaks(const float* frames, std::size_t frameCount,
```

Cppcheck (2.20.0)

[style] The function 'reducePeaks' is never used.

(unusedFunction)